types/mercadopago-sdk-js/package.json

```diff
@@ -11,10 +11,5 @@
     "devDependencies": {
         "@types/mercadopago-sdk-js": "workspace:."
     },
-    "owners": [
-        {
-            "name": "mercadopago",
```

Comment:
still seems wrong. github.com/mercadopago exists and makes sense to own a package named mercadopago-sdk-js